File: CloudFlow/FrontEnd-App/src/app/shared/model/move-folder-request.model.ts
import { JsonProperty, JsonObject } from "json2typescript";

@JsonObject
export class MoveFolderRequestModel {
	
	@JsonProperty("folderToMoveId", String)
	public folderToMoveId: string = null;
	
	@JsonProperty("destinationFolderId", String)
	public destinationFolderId: string = null;
	
	@JsonProperty("documentLocationRealName", String)
	public documentLocationRealName: string = null;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/content/MoveFolderRequestModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.content;

public class MoveFolderRequestModel {
	
	private String folderToMoveId;
	private String destinationFolderId;
	private String documentLocationRealName;
	
	public String getFolderToMoveId() {
		return folderToMoveId;
	}
	public void setFolderToMoveId(String folderToMoveId) {
		this.folderToMoveId = folderToMoveId;
	}
	public String getDestinationFolderId() {
		return destinationFolderId;
	}
	public void setDestinationFolderId(String destinationFolderId) {
		this.destinationFolderId = destinationFolderId;
	}
	public String getDocumentLocationRealName() {
		return documentLocationRealName;
	}
	public void setDocumentLocationRealName(String documentLocationRealName) {
		this.documentLocationRealName = documentLocationRealName;
	}
}
